packages/functional-tests/tests/subscription-tests/coupons-ui-tests/resubscription.spec.ts

```diff
@@ -126,7 +126,9 @@ test.describe('resubscription test', () => {
     expect(await subscriptionManagement.getCardInfo()).toContain('4444');
   });
 
-  test('update mode of payment for paypal', async ({
+  //Diabling the test as this is being flaky because Paypal Sandbox is being finicky
+  // FXA - 6786
+  /*test('update mode of payment for paypal', async ({
```

Comment:
This works. Not sure it should be `.skip()` instead, but whatever. Let's YOLO this in and argue later.